lib/dor/datastreams/content_metadata_ds.rb

```diff
@@ -95,10 +97,12 @@ def to_solr(solr_doc = {}, *args)
           counts['content_file'] += 1
           preserved_size += file['size'].to_i if file['preserve'] == 'yes'
           shelved_size += file['size'].to_i if file['shelve'] == 'yes'
-          next unless file['shelve'] == 'yes'
-          counts['shelved_file'] += 1
-          first_shelved_image ||= file['id'] if file['id'] =~ /jp2$/
+          if file['shelve'] == 'yes'
+            counts['shelved_file'] += 1
+            first_shelved_image ||= file['id'] if file['id'] =~ /jp2$/
+          end
           mime_types << file['mimetype']
```

Comment:
It looks like we were skipping adding the file's mimetype to the `mime_types` set before this change.  Is it an intentional change to add un-shelved file's mimetypes here?

Reply:
Yes; I caught this when making the change. It's in a separate commit (3a50dd8) than the `@role` attribute work.

Reply:
Ah, didn't notice the separate commit.  Thanks.